### PR TITLE
Fix missing translations

### PR DESCRIPTION
**Pull request by armandfardeau**

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -58,44 +57,75 @@ fr:
...
+      update:
+        error: fr.decidim.initiatives.update.error
+        success: fr.decidim.initiatives.update.success
+    omniauth:
+      france_connect:
+        external:
+          link: https://franceconnect.gouv.fr/
+          text: Qu'est-ce-que FranceConnect ?
+      france_connect_profile:
+        explanation: '<b>L''auteur d''une pétition ne peut être anonyme.</b><br/>
```
Comment (Quentinchampenois) [inline config/locales/fr.yml:117]: Hi, 

Could you fix the double simple quotes in values like this one  for example ?
```
L''auteur d''une pétition ne peut être anonyme.
```

Thanks !

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]

File: config/locales/fr.yml (hunk 1 above)
Comment (Quentinchampenois) [reply config/locales/fr.yml:117]: To be more precise, I didn't check in local but I think they won't be escaped

**Comment by paulinebessoles** [comment]
@armandfardeau could you review the PR so it can be merged?

File: config/locales/fr.yml (hunk 1 above)
Comment (Quentinchampenois) [reply config/locales/fr.yml:117]: I have tested locally, I apologize it works well.

Seems good to me !

**Review by Quentinchampenois — APPROVED** [review]

**Pull request merged**